src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -1,377 +1,353 @@
-        var {columns, data} = fullRawTable.tableData;
-        var tIdx = columns.findIndex((col) => (col.name === get(layoutInfo, [LC.MISSION_DATA, LC.META_TIME_CNAME])));
-        var arr = data.reduce((prev, e)=> {
-            prev.push(parseFloat(e[tIdx]));
-            return prev;
-        }, []);
-
-        var [tzero, tzeroMax] = arr.length > 0 ? [Math.min(...arr), Math.max(...arr)] : [0.0, 0.0];
-        var max = 365;
-        var min = Math.pow(10, -3);   // 0.001
-
-        // var period = get( FieldGroupUtils.getGroupFields(LC.FG_PERIOD_FINDER), ['period', 'value'], '');
-
-        const period = '';
-        var fields = FieldGroupUtils.getGroupFields(LC.FG_PERIOD_FINDER);
-        var initState;
-
-        if (fields) {      // fields already exists and new table is loaded
-            initState = [
-                {fieldKey: 'time', value: get(layoutInfo, [LC.MISSION_DATA, LC.META_TIME_CNAME])},
-                {fieldKey: 'flux', value: get(layoutInfo, [LC.MISSION_DATA, LC.META_FLUX_CNAME])},
-                {fieldKey: 'periodMin', value: `${min}`},
-                {fieldKey: 'periodMax', value: `${max}`},
-                {fieldKey: 'period', value: `${period}`},
-                {fieldKey: 'tzero', value: `${tzero}`},
-                {fieldKey: 'tzeroMax', value: `${tzeroMax}`}];
-
-            dispatchMultiValueChange(LC.FG_PERIOD_FINDER, initState);
-        }
-        fields = FieldGroupUtils.getGroupFields(LC.FG_PERIODOGRAM_FINDER);
-        if (fields) {
-            dispatchRestoreDefaults(LC.FG_PERIODOGRAM_FINDER);
-        }
-
-        //IRSA-464: the smartMerge does not replace array component.  It updates the array. Therefore, setting it
-        //null first to force the new fullRawTable is updated to the layout
-        dispatchUpdateLayoutInfo(Object.assign({}, layoutInfo, {
-            fullRawTable:null,
-            periodRange: {min, max, tzero, tzeroMax, period}
-        }));
-        //add the fullRawtable into the layout
-        dispatchUpdateLayoutInfo(Object.assign({}, layoutInfo, {
-            fullRawTable,
-            periodRange: {min, max, tzero, tzeroMax, period}
-        }));
-
-
-        callback && callback();
-    };
-
-    var rawTable = getTblById(LC.RAW_TABLE);
-    if (layoutInfo.fullRawTable && layoutInfo.fullRawTable.totalRows===rawTable) {
-        callback && callback();
-    } else {
-
-        if (isTblDataAvail(0, rawTable.totalRows, rawTable)) {
-            setTableData(rawTable);
-        } else {
-            var req = Object.assign(cloneDeep(rawTable.request), {pageSize: MAX_ROW});
-
-            doFetchTable(req).then(
-                (tableModel) => {
-                    setTableData(tableModel);
-                }
-            ).catch(
-                (reason) => {
-                    logError(`Failed to get full raw table: ${reason}`, reason);
-                }
-            );
-       }
-    }
-}
+/*

```

Comment:
What happen here?  Why github thinks you've changes to the whole file?  Are you sure it's not some sort of git related issues?  Double check to make sure you don't override previous changes.